tests/nested/hot-plug/task.yaml

```diff
@@ -1,19 +1,15 @@
 summary: create ubuntu classic image, install snapd and test hot plug feature
 
-systems: [ubuntu-16.04-64, ubuntu-18.04-64]
-
 prepare: |
-
     "$TESTSLIB"/prepare-restore.sh --prepare-suite
     "$TESTSLIB"/prepare-restore.sh --prepare-suite-each
 
     #shellcheck source=tests/lib/nested.sh
     . "$TESTSLIB/nested.sh"
     create_nested_classic_vm
     copy_remote "${GOHOME}"/snapd_*.deb
-    execute_remote 'sudo apt update'
-    execute_remote 'sudo apt install -y ./snapd_*.deb'
-    #shellcheck disable=SC2016
+    execute_remote "sudo apt update"
+    execute_remote "sudo apt install -y ./snapd_*.deb"
     execute_remote 'sudo apt install -y linux-image-extra-$(uname -r) || sudo apt install -y linux-modules-extra-$(uname -r)'
```

Comment:
Can linux-image-extra be pre-installed on the custom image now and removed here?

Reply:
With the image we are using now it is not possible, in the near future I'll prepare a new image wich will include a nested vm inside and all its dependencies. So far the change I did is to start using this new image which is configured to run faster with virtualization.